.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: enabled
 contact_links:
   - name: Cryostat Community Support
-    url: https://github.com/cryostatio/cryostat/discussions
+    url: https://github.com/cryostatio/cryostat3/discussions
```

Comment:
So far the plan has been to rename the `cryostatio/cryostat` repo to something else (TBD)[1], and rename `cryostatio/cryostat3` to take the place of `cryostatio/cryostat`. This way the new codebase *should*, if all goes well, drop into place and all existing links like these will just point to the new sources.

[1] Maybe `cryostat-2.x`? That's the same pattern I used when forking the web client. It might be confusing though, since people might assume that `-2.x` is more recent than the one with no suffix.

Reply:
Perhaps, but if we archive it and add a redirection note on the top of the README, it'll probably be okay, no?

Reply:
I hope so. That's the plan anyway.

https://github.com/cryostatio/cryostat/pull/1934

(also needs updating once the repos are renamed)

Reply:
Any better ideas than `cryostat-2.x`? `cryostat-old`, `cryostat-classic`?

Reply:
`cryostat-legacy`?

Reply:
Legacy seems like a good description of it. I'll go with that for now, but Elliott or Chris should be able to rename it again while I'm away on PTO if the team decides to go with something else.

Reply:
https://github.com/cryostatio/cryostat-legacy/

Reply:
On the other side, https://github.com/cryostatio/cryostat should now point to what was https://github.com/cryostatio/cryostat3 (which should redirect to the first link), aka the current development 3.0 project.